AEPCampaignClassic/Sources/CampaignClassicConstants.swift

```diff
@@ -34,6 +34,12 @@ enum CampaignClassicConstants {
         static let TOKEN_HASH = "ADOBEMOBILE_STOREDDEFAULTS_TOKENHASH"
         static let REGISTER_STATUS = "ADOBEMOBILE_STOREDDEFAULTS_REGISTERSTATUS"
     }
+    
+    
+    enum SDKEventType {
```

Comment:
why not just `EventType`?

Reply:
Coming from the world of Assurance, I had to differentiate between `AssuranceEventType` and `SDKEventType`
Does need to be here 

Changed to `EventType`